satellite/satellitedb/customers.go

```diff
@@ -0,0 +1,30 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package satellitedb
+
+import (
+	"context"
+
+	"github.com/skyrings/skyring-common/tools/uuid"
+
+	dbx "storj.io/storj/satellite/satellitedb/dbx"
+)
+
+// customers is an implementation of payments.Accounts
+type customers struct {
+	db *dbx.DB
+}
+
+// Insert is a method for inserting stripe customer into the database.
+func (customers *customers) Insert(ctx context.Context, userID uuid.UUID, customerID string) (err error) {
+	defer mon.Task()(&ctx)(&err)
+
+	_, err = customers.db.Create_StripeCustomers(
+		ctx,
+		dbx.StripeCustomers_UserId(userID[:]),
```

Comment:
```suggestion
		dbx.StripeCustomers_UserId([]byte(userID)),
```

Reply:
we have such syntax in all our repos. I would like not to break consistency

Reply:
Sorry, my fault, I confused the type of `userID`, I bore in mind that was a String but it's a UUID.